index.bs

```diff
@@ -63,6 +65,7 @@ Some user agents defer to their host OS to manage and secure crypto key material
 1. Untrusted script can launch denial-of-service attacks against the user's keystore, since form submission (including keygen) does not require user action.
 2. Keygen form submission has unmitigated access to the keystore; script can affect either 1) global OS state changes without user permission, or 2) origin-unaware certificate installation (in user agents that implement their own keystore). At a minimum, this violates the same origin policy.
 3. While not specifically impacting keygen, user agents may automatically install client certificates via a special mime-type. As noted, this can affect state outside the boundary of the browser sandbox without user permission.
+4. The primary identification of a certificate (as shown to the user in step 1 of the authentication process) is that by ether most of the Distingished Name (DN) or by just its (first) element, the Common Name. When a certificate is created (with the users public key to be subsequently signed) the DN is under (full) control of the server. It is therefore possible, especially if the previous step '3' is weak; for an adversary to put an identical (or credible) looking entry into the pull-down choise menu of the user. Often browsers allow control of the order (through the expire date; also set by the server). This makes it easy to trick a user in using the wrong certificate or to 'spring clean' his real certificate.
```

Comment:
I think this boils down to phishing the user. I like this, but this sentence is very wordy and complex. Can you reduce/simplify it?